conf/broker.yaml.template

```diff
@@ -1,7 +1,5 @@
 BROKER:
-    # The path where your broker settings and inventory are located
-    # If you leave it blank, the default is the output of `broker --version`
-    BROKER_DIRECTORY:
+    # Broker has its own config which you can find by running `broker --version`
```

Comment:
Many of us generally likes to keep conf files in robottelo root directory, so what if we could have made a broker to choose the conf file path rather than keeping it at fixed location.

Reply:
you can do that today, but it still wouldn't work correctly with robottelo setting it. just `export BROKER_DIRECTORY=/path/to/robottelo`